scripts/generate-utilities-docs.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+const mustache = require("mustache");
+const fs = require("node:fs");
+const childProcess = require("node:child_process");
```

Comment:
Optional nit: this looks fine but when internal Node modules are used elsewhere in the project, we drop the `node:` (which is optional) and use object deconstruction in the imports:

```suggestion
const { readFileSync, writeFileSync } = require('fs');
const { execSync } = require('child_process');
```

This is slightly more semantic than *purely* stylistic so again, I'm happy to keep it as is! Just wanted to make a note for consistency's sake.